pyproject.toml

```diff
@@ -25,7 +25,7 @@ pytest-mock = "^3.4.0"
 pre-commit = {version = "^2.9.3", python = "^3.6.1"}
 pytest-cov = "^2.10.1"
 tox = "^3.20.1"
-black = "^20.8b1"
+black = "22.3.0"  # exact version as in pre-commit
```

Comment:
we've actually been removing linting dependencies from `pyproject.toml` across our projects so we don't have this duplication. can you go ahead and do that here?

```suggestion
```

as well as reverting the `python` constraint 🙏

Reply:
Ah... ok. So frustrated to hear that though. Is there a discussion where I can share my frustrations and alternative ideas?

Reply:
We have a tracking issue for our linting efforts at https://github.com/python-poetry/poetry/issues/4776, but we haven't had much discussion there. You could also start a new discussion on [GitHub](https://github.com/python-poetry/poetry/discussions) or join our [discord](https://github.com/python-poetry/poetry#resources)